pulp_deb/app/serializers/content_serializers.py

```diff
@@ -273,6 +273,13 @@ def from822(cls, data, **kwargs):
                 # also save the fields not in TRANSLATION_DICT
                 args["custom_fields"][k] = v
 
+        # Delete package fields with corrupt values
+        del args['installed_size'] if 'installed_size' in args and not isinstance(args['installed_size'], int)
+        del args['essential'] if 'essential' in args and not isinstance(args['essential'], bool)
+        del args['build_essential'] if 'build_essential' in args and not isinstance(args['build_essential'], bool)
+        if 'multi_arch' in args and not isinstance(args['multi_arch'], str):
```

Comment:
If I am right about all fields containing strings at this point then we don't need this test:
```suggestion
        if 'multi_arch' in args:
```